index.js

```diff
@@ -0,0 +1,57 @@
+var express = require("express");
+var hbs = require("express-handlebars");
+
+var app = express();
+
+var compliments = [
+  "Your instructors love you",
+  "High five = ^5",
+  "Is it Ruby Tuesday yet?",
+  "It's almost beer o'clock",
+  "The Force is strong with you"
+];
+
+
+var compliment = function(){
+  var randomNumber = Math.random();
+
+  if (randomNumber < .2){
+    return compliments[0]
+  } else if (randomNumber < .4){
+    return compliments[1]
```

Comment:
Great job here, you could also use `var randomIndex = Math.floor(Math.random()*compliments.length)]`